api/src/main/java/com/github/eyefloaters/console/api/errors/client/ConstraintViolationHandler.java

```diff
@@ -1,4 +1,4 @@
-package com.github.eyefloaters.console.api.mapping;
+package com.github.eyefloaters.console.api.errors.client;
```

Comment:
If we're adding subpackages lets add them with a `package-info.java` to make explicit the rationale for what belongs where. That makes it much easer for later contributors to figure out for themselves where any new handler would belong.

Reply:
Added `package-info.java` files with JavaDoc comments :+1: